src/index.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+// const axios = require('axios');
+const state = {
+  city: '',
+};
+
+function incrementButton() {
+  const element = document.getElementById('counter');
+  let value = element.innerHTML;
+
+  value++;
+
+  temperatureColor(value);
+  landscapeIcons(value);
+
+  document.getElementById('counter').innerHTML = value;
+}
+
+function decreaseButton() {
+  const element = document.getElementById('counter');
+  let value = element.innerHTML;
+
+  value--;
+
+  temperatureColor(value);
+  landscapeIcons(value);
+
+  document.getElementById('counter').innerHTML = value;
+}
```

Comment:
These work great! Can you consider how you could make your code more DRY by combining them into a single function?